components/lib/crash/src/test/java/mozilla/components/lib/crash/BreadcrumbPriorityQueueTest.kt

```diff
@@ -33,49 +33,41 @@ class BreadcrumbPriorityQueueTest {
     }
 
     @Test
-    fun `Breadcrumb priority queue stores the correct priority`() {
+    fun `Breadcrumb priority queue stores the latest breadcrumbs`() {
         val testMessage = "test_Message"
         val testData = hashMapOf("1" to "one", "2" to "two")
         val testCategory = "testing_category"
         val testType = Breadcrumb.Type.USER
-        val maxNum = 5
+        val maxNum = 10
         val crashBreadcrumbs = BreadcrumbPriorityQueue(maxNum)
 
         repeat(maxNum) {
             crashBreadcrumbs.add(
-                    Breadcrumb(testMessage, testData, testCategory, Breadcrumb.Level.DEBUG, testType)
-            )
-        }
-        assertEquals(crashBreadcrumbs.size, maxNum)
-
-        repeat(maxNum) {
-            crashBreadcrumbs.add(
-                    Breadcrumb(testMessage, testData, testCategory, Breadcrumb.Level.INFO, testType)
+                Breadcrumb(testMessage, testData, testCategory, Breadcrumb.Level.CRITICAL, testType)
             )
+            sleep(10) /* make sure time elapsed */
```

Comment:
Ideally we'd mock the time to not actually slow down our tests. But it's not much, so probably fine to land.